inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -113,16 +113,18 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
   :root {
     --bslib-dashboard-sidebar-bg: var(--#{$prefix}body-bg);
     --bslib-dashboard-sidebar-main-bg: var(--#{$prefix}body-bg);
+    --bslib-dashboard-main-bg: rgb(247, 247, 247);
     --#{$prefix}card-border-color: var(--bslib-dashboard-border-color-translucent, var(--#{$prefix}border-color-translucent));
     --bslib-sidebar-bg: var(--#{$prefix}body-bg);
     --bslib-sidebar-fg: var(--#{$prefix}color);
   }
 
+  @include color-mode(dark) {
+    --bslib-dashboard-main-bg: rgb(20, 20, 24);
+  }
+
   // Bootstrap sets the body bg color, but dashboard style may make the body bg
   // transparent, so we need to set the html bg color as well
-  html {
-    background-color: var(--bs-body-bg);
-  }
```

Comment:
It turns out this approach doesn't work in Firefox or Safari, where we really need to set the background color of the `<body>` tag.